certification/internal/engine/engine.go

```diff
@@ -93,6 +99,19 @@ func (c *CraneEngine) ExecuteChecks() error {
 		return fmt.Errorf("%w: %s", errors.ErrExtractingTarball, err)
 	}
 
+	// if --submit flag is set write json files to disk to be used to send to pyxis after all checks are complete
+	if viper.GetBool("submit") {
+		err := writeCertImage(img)
+		if err != nil {
+			return err
+		}
+
+		err = writeRPMManifest(containerFSPath)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
I feel this block should be placed after the check execution. I am unsure of the usecase; can a partner submit a result even though some checks fail?

Reply:
Two things:
- It is here because `crane` has already pull/extracted the image on disk, if we do it later, we would have to pull/extract again. I wanted to make sure it was before any of the defer's were called.
- Yes, they can submit failed results, preflight is not validating the output, we submit the raw results to pyxis.